Description: skip database tests if PERL_SKIP_DB_TESTS is in the environment
 Testing with a real database requires a running firebird server. This cannot
 be ensured with the current state of the Debian packages, so we make possible
 that the tests are skipped when the environment contains the
 PERL_SKIP_DB_TESTS variable.
 This is used to skip database testing during debian package build.
Author: Damyan Ivanov <dmn@deban.org>
Forwarded: no

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -187,6 +187,7 @@ sub make_test_conf
 
     # ask for database path
     DBPATH: {
+        last DBPATH if $ENV{PERL_SKIP_DB_TESTS};
         for (1..3) {
             last if $path = prompt("\nFull path to test database to create or use: ", $path);
         }
